### ISSUE TITLE
Syntax error

### ISSUE THREAD

**Issue by H1net** [matheuss/hpm#4]
```
[h1net@DFN80MAC .hyperterm_plugins]$ hpm install hyperpower
/usr/local/lib/node_modules/hpm-cli/index.js:10
updateNotifier({pkg}).notify();
                   ^
SyntaxError: Unexpected token }
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
[h1net@DFN80MAC .hyperterm_plugins]$ 
```

**Comment by Leeds-eBooks** [comment]
Also here:
```
/Users/ben/.nvm/versions/node/v4.0.0/lib/node_modules/hpm-cli/index.js:18
        let msg = chalk.red('You don\'t have HyperTerm installed! :(\n');
        ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:413:25)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
    at startup (node.js:117:18)
    at node.js:951:3
```
I don't know how hyperterm handles ES6 but I think it's not transpiling automatically, so this may depend on the version of node.js you have installed.

**Comment by Leeds-eBooks** [comment]
Ah this is a dup of #1

**Comment by H1net** [comment]
Ah yes indeed. Thanks @Leeds-eBooks

**Comment by matheuss** [comment]
Thanks guys! Fixed in [v1.0.0-RC1](https://github.com/matheuss/hpm/tree/v1.0.0-RC1) 😊
